### DISCUSSION TITLE
Should I add *.freezed.dart files to .gitignore?

### DISCUSSION THREAD

**Discussion by proshin-roman** [rrousselGit/freezed#943] [Q&A] [answered]
I couldn't find any "official" information about that question, as well as nothing on Google.

Hence, here is my question: 

Should I add `*.freezed.dart` files to `.gitignore` so that they are always generated from scratch?

**Comment by rrousselGit** [comment, marked as answer]
That's up to you. There's no official answer

**Reply by proshin-roman** [reply]
thank you @rrousselGit for the answer! :)
